browser/sources.gni

```diff
@@ -132,6 +132,7 @@ brave_chrome_browser_deps = [
   "//brave/browser/ui",
   "//brave/browser/ui/tabs",
   "//brave/common",
+  "//brave/components/ai_chat/content/browser",
```

Comment:
and this under `if (ai_chat_enabled)`

Reply:
thanks, fixed at 
https://github.com/brave/brave-core/pull/20996/commits/b46d3237a9a659589182e66c39a9f573030885c4#diff-d00e7b939f2d684a6f82e428ab699596962b739950acc1da8804abb5f607d1b3L135

this dependency  already was at https://github.com/brave/brave-core/blob/master/browser/sources.gni#L289